Add wake_word integration docs

synesthesiam opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Document new `wake_word` integration: https://github.com/home-assistant/core/pull/96380
- can be merged once text fragment is merged: #29129


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

synesthesiam committed:
Add wake_word integration docs

c0ffeeca7 reviewed (dismissed):
LGTM, @synesthesiam :+1:

synesthesiam commented:
Please wait to merge until https://github.com/home-assistant/core/pull/96380 has been merged

synesthesiam committed:
Bump HA version

synesthesiam committed:
Include openWakeWord addon

synesthesiam commented:
Ready to merge

frenck commented on source/_integrations/wake_word.markdown:
```diff
@@ -0,0 +1,19 @@
+The wake-word-detection entities cannot be implemented manually, but can be provided by integrations such as [Wyoming](/integrations/wyoming). The API in [Assist Pipelines](https://developers.home-assistant.io/docs/voice/pipelines/) enables wake-word-detection as part of [Assist](/voice_control/).
+
+
+{% include integrations/config_flow.md %}
```
This cannot be set up like that for entity components?

c0ffeeca7 replied:
```suggestion

{% include integrations/building_block_integration.md %}
```

c0ffeeca7 committed:
Replace config flow by building block integration text fragment

informing that this integration is used by other integrations as a building block. It can not / does not have to be set up using the UI or YAML config flow. It is installed automagically by an other integration, if it is used.
